core/vm/gas.go

```diff
@@ -123,61 +123,64 @@ type req struct {
-	JUMP:           {1, GasMidStep, 0},
-	JUMPI:          {2, GasSlowStep, 0},
-	EXP:            {2, GasSlowStep, 1},
-	ADDRESS:        {0, GasQuickStep, 1},
-	ORIGIN:         {0, GasQuickStep, 1},
-	CALLER:         {0, GasQuickStep, 1},
-	CALLVALUE:      {0, GasQuickStep, 1},
-	CODESIZE:       {0, GasQuickStep, 1},
-	GASPRICE:       {0, GasQuickStep, 1},
-	COINBASE:       {0, GasQuickStep, 1},
-	TIMESTAMP:      {0, GasQuickStep, 1},
-	NUMBER:         {0, GasQuickStep, 1},
-	CALLDATASIZE:   {0, GasQuickStep, 1},
-	DIFFICULTY:     {0, GasQuickStep, 1},
-	GASLIMIT:       {0, GasQuickStep, 1},
-	POP:            {1, GasQuickStep, 0},
-	PC:             {0, GasQuickStep, 1},
-	MSIZE:          {0, GasQuickStep, 1},
-	GAS:            {0, GasQuickStep, 1},
+	ADD:          {2, GasFastestStep, 1},
+	LT:           {2, GasFastestStep, 1},
+	GT:           {2, GasFastestStep, 1},
+	SLT:          {2, GasFastestStep, 1},
+	SGT:          {2, GasFastestStep, 1},
+	EQ:           {2, GasFastestStep, 1},
+	ISZERO:       {1, GasFastestStep, 1},
+	SUB:          {2, GasFastestStep, 1},
+	AND:          {2, GasFastestStep, 1},
+	OR:           {2, GasFastestStep, 1},
+	XOR:          {2, GasFastestStep, 1},
+	NOT:          {1, GasFastestStep, 1},
+	BYTE:         {2, GasFastestStep, 1},
+	SHL:          {2, GasFastestStep, 1},
+	SHR:          {2, GasFastestStep, 1},
+	SAR:          {2, GasFastestStep, 1},
+	CALLDATALOAD: {1, GasFastestStep, 1},
+	CALLDATACOPY: {3, GasFastestStep, 1},
+	MLOAD:        {1, GasFastestStep, 1},
+	MSTORE:       {2, GasFastestStep, 0},
+	MSTORE8:      {2, GasFastestStep, 0},
+	CODECOPY:     {3, GasFastestStep, 0},
+	MUL:          {2, GasFastStep, 1},
+	DIV:          {2, GasFastStep, 1},
+	SDIV:         {2, GasFastStep, 1},
+	MOD:          {2, GasFastStep, 1},
+	SMOD:         {2, GasFastStep, 1},
+	SIGNEXTEND:   {2, GasFastStep, 1},
+	ADDMOD:       {3, GasMidStep, 1},
+	MULMOD:       {3, GasMidStep, 1},
+	JUMP:         {1, GasMidStep, 0},
+	JUMPI:        {2, GasSlowStep, 0},
+	EXP:          {2, GasSlowStep, 1},
+	ADDRESS:      {0, GasQuickStep, 1},
+	ORIGIN:       {0, GasQuickStep, 1},
+	CALLER:       {0, GasQuickStep, 1},
+	CALLVALUE:    {0, GasQuickStep, 1},
+	CODESIZE:     {0, GasQuickStep, 1},
+	GASPRICE:     {0, GasQuickStep, 1},
+	COINBASE:     {0, GasQuickStep, 1},
+	TIMESTAMP:    {0, GasQuickStep, 1},
+	NUMBER:       {0, GasQuickStep, 1},
+	CALLDATASIZE: {0, GasQuickStep, 1},
+	DIFFICULTY:   {0, GasQuickStep, 1},
+	GASLIMIT:     {0, GasQuickStep, 1},
+	POP:          {1, GasQuickStep, 0},
+	PC:           {0, GasQuickStep, 1},
+	MSIZE:        {0, GasQuickStep, 1},
+	GAS:          {0, GasQuickStep, 1},
+	BLOCKHASH:    {1, GasExtStep, 1},
+	BALANCE:      {1, new(big.Int), 1},
+	EXTCODESIZE:  {1, new(big.Int), 1},
+	EXTCODECOPY:  {4, new(big.Int), 0},
+	SLOAD:        {1, big.NewInt(50), 1},
+	SSTORE:       {2, new(big.Int), 0},
+	SHA3:         {2, big.NewInt(30), 1},
+	CREATE:       {3, big.NewInt(32000), 1},
 	RETURNDATASIZE: {0, GasQuickStep, 1},
```

Comment:
I know this is a small nitpick but can you move this line to below the GAS opcode to keep the GasQuickStep costs together and format the code